src/tools/interfaces.md

```diff
@@ -1,38 +1,41 @@
-Mozilla Firefox Data Analysis Tools
------------------------------------
+# Tools for Data Analysis
 
-This is a starting point for making sense of (and gaining access to) all of the
-Firefox-related data analysis tools. There are a number of different tools
-available, all with their own strengths, tailored to a variety of use cases and
-skill sets.
+This is a starting point for making sense of the tools used for analyzing Mozilla data. There are different tools available, each with their own strengths, tailored to a variety of use cases and skill sets.
 
-#### [Mozilla Growth & Usage Dashboard](https://growth-stage.bespoke.nonprod.dataops.mozgcp.net/) (GUD)
+<!-- toc -->
 
-The Mozilla Growth & Usage Dashboard (GUD) is a tool to visualize growth metrics in a standard way across Mozilla’s products. It is designed to be intuitive to use and does not assume any in-depth technical knowledge: this is the first place you should look if you have a question like "how many people are using X?".
+## High-level tools
 
-#### [`sql.telemetry.mozilla.org`](stmo.md) (STMO)
+These web-based tools do not require specialized technical knowledge (e.g. how to write an SQL query, deep knowledge of BigQuery). This is where you should start.
+
+### Mozilla Growth & Usage Dashboard (GUD)
+
+The [Mozilla Growth & Usage Dashboard](<(https://growth-stage.bespoke.nonprod.dataops.mozgcp.net/)>) (GUD) is a tool to visualize growth metrics in a standard way across Mozilla’s products. This is the first place you should look if you have a question like "how many people are using X?".
+
+### Glean Aggregated Metrics Dashboard (GLAM)
+
+The [Glean Aggregated Metrics Dashboard](https://glam-stage.bespoke.nonprod.dataops.mozgcp.net/) (GLAM) is an interactive dashboard that is Mozilla’s primary self-service tool for examining the distributions of values of specific individual telemetry metrics, over time and across different user populations. It is similar to GUD in that it is meant to be usable by everyone; no specific data analysis or coding skills are needed. But while GUD is focused on a relatively small number of high level, derived product metrics about user engagement (e.g. MAU, DAU, retention, etc) GLAM is focused on a diverse and plentiful set of probes and data points that engineers capture in code and transmit back from Firefox and other Mozilla products.
```

Comment:
@rafrombrc @hamilton @openjck @spasovski does this seem like a reasonable description for GLAM?

Reply:
Yep! This is an almost verbatim copy of [the documentation that we link to from the GLAM home page](https://docs.google.com/document/d/1qIkEDemnODbYuVIfpciohgEXcaFjrK_mfVG5FOSeVuM/preview). Out of curiosity, did you find it somewhere else?

Reply:
Heh yup, I should have mentioned the source. 😃